Lib/test/test_tkinter/test_widgets.py

```diff
@@ -354,6 +354,10 @@ def test_bad_kwarg(self):
         with self.assertRaisesRegex(TclError, r"^unknown option -image$"):
             tkinter.OptionMenu(self.root, None, 'b', image='')
 
+    def test_specify_name(self):
+        widget = tkinter.OptionMenu(self.root, None, ':)', name="option_menu")
+        self.assertIs(self.root.nametowidget("option_menu"), widget)
```

Comment:
Interesting. I expected using `str()` for test, but `nametowidget()` works too.

I would suggest to use `children`, because `nametowidget()` is not documented (and not even tested!).

```suggestion
        self.assertEqual(str(widget), ".option_menu")
        self.assertIs(self.root.children["option_menu"], widget)
```